tst/hashfunctions/recursive.tst

```diff
@@ -53,5 +53,10 @@ gap> HashBasic(rec1) = HashBasic(rec2);
 true
 
 # Check objects we don't handle
+#@if GAPInfo.KernelInfo.KERNEL_API_VERSION >= 7000
```

Comment:
But isn't this feature new in GAP 4.11? I.e., won't help here

Reply:
I went through all these ideas when I looked at the test failures after they were reported. Only solutions I could come up with:

1. drop the test,
2. adjust our error function to not use TNAM_OBJ directly but rather our own wrapper func which handles T_POSOBJ (and perhaps more) special (and uniformly across all GAP versions).
3. move the problematic tests to a separate .tst file, and then use the 'exclude' feature of TestDirectory to only run it in GAP >= 4.11

Reply:
I'm tempted to drop the test, as I'm not sure it's worth the pain of the other options

Reply:
I agree and made PR #127 to do just that.